develop/tutorials/articles/220-internationalization/01-localizing-your-application.markdown

```diff
@@ -197,10 +206,12 @@ bundle.
 
         Provide-Capability: liferay.resource.bundle;resource.bundle.aggregate:String="(bundle.symbolic.name=com.liferay.my.editor.lang)";resource.bundle.base.name="content.Language"
```

Comment:
@jhinkey I got really confused at this line the first time I went through it because the bundle.symbolic.name didn't match the one from the last bullet point. This one says com.liferay.my.editor.lang